x-pack/plugins/observability/public/pages/overview/old_overview_page.tsx

```diff
@@ -94,6 +98,14 @@ export function OverviewPage({ routeParams }: Props) {
     refetch.current = ref;
   }, []);
 
+  const handleGuidedSetupClick = useCallback(() => {
+    if (isGuidedSetupHidden) {
+      trackMetric({ metric: 'guided_setup_view_details_after_dismiss' });
```

Comment:
What does the `_after_dismiss` part of this metric name mean?

Reply:
we have two ways of showing the flyout with the guided setup: through the "Guided setup" button at the header (1)(handled by this callback) and through a blue panel bar (2). This panel can be dismissed (3), and once dismissed it doesn't show up anymore.

<img width="1300" alt="Screenshot 2022-03-28 at 17 19 46" src="https://user-images.githubusercontent.com/57448/160431523-bc2e9074-02b9-4cda-85b1-963b28526ed3.png">

What we want to measure is how many people open the flyout through the "Guided setup" button _after_ the blue panel has been dismissed.